app/controllers/api/v0/admins_controller.rb

```diff
@@ -81,6 +120,20 @@ def fetch_workflows
         json_response(workflows)
       end
 
+      def group_operation
+        group = Group.find(params.require(:id))
+        group.send(group_params.require(:operation), group_params.require(:approver_ids))
```

Comment:
will this work? `:approver_id` is not a direct child of `group_params`. You should introduce a service class.

Reply:
`parameters` is the keyword in controller. We need to choose  another word to replace it.

Reply:
arguments?

Reply:
or params?